### PR TITLE
Add Socialblade Markdown

### PR DESCRIPTION
**Pull request by meauxt**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14060

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by fabaff** [comment]
This PR should only contain the docs for the Social Blade sensor.

File: source/_components/sensor.socialblade.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+The channel id will be the last part of the url :
+https://socialblade.com/youtube/channel/{channel_id}
+
+## Example for `configuration.yaml` :
+
+```yaml
+sensor:
+  - platform: socialblade
+    channel_id: YOUR_CHANNEL_NUMBER
+    name: OPTIONAL_NAME
```
Comment (fabaff) [inline source/_components/sensor.socialblade.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/sensor.socialblade.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+{% configuration %}
+friendly_name:
```
Comment (fabaff) [inline source/_components/sensor.socialblade.markdown]: Should be `name:`.

File: source/_components/sensor.socialblade.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  required: false
+  default: Social Blade
+  type: string
+channel_id:
+  description: channel id number optained from the URL when you access Social Blade web client. 
+  required: true
+  type: string
+{% endconfiguration %}
+
+All the data will be fetch from  [Social Blade]( https://socialblade.com/) using an [unofficial Social Blade Client]( https://pypi.org/project/socialbladeclient/).
```
Comment (fabaff) [inline source/_components/sensor.socialblade.markdown]: There is no need to mention the used client as this is transparent for the user.

**Comment by fabaff** [comment]
@meauxt, can you please remove the `uscis` stuff. Thanks.

**Comment by meauxt** [comment]
@fabaff  USCIS stuff has been removed.

**Review by balloob — APPROVED** [review]

**Pull request merged**